.github/workflows/build-on-windows.yml

```diff
@@ -0,0 +1,286 @@
+        run: |
+          cmake -G "Visual Studio 16 2019" -A ${{ env.PLATFORM }}                           ^
+                -S ${{ github.workspace }} -B ../build                                      ^
+                -DCMAKE_BUILD_TYPE=%BUILD_TYPE%                                             ^
+                -DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake            ^
+                -DVCPKG_TARGET_TRIPLET=${{ matrix.config.VCPKG_TARGET_TRIPLET }}            ^
+                -DBUILD_SHARED_LIBS=${{ matrix.config.BUILD_SHARED_LIBS }}                  ^
+                -DTT_BUILD_PCH=${{ matrix.config.TT_BUILD_PCH }}                            ^
+                -DCMAKE_VERBOSE_MAKEFILE=ON
+
+      - name: 🙏 CMake ➔ Build
+        run: |
+          cd ..\build
+          cmake --build . --config %BUILD_TYPE% -j %NUMBER_OF_PROCESSORS%
+
+      - name: 📦 CMake ➔ Install
+        run: cmake --install ..\build --config %BUILD_TYPE% --prefix ..\install --strip --verbose
+
+      - name: ❔ CHECK important folders, to see if everything is present (after building)
+        run: |
+          dir ..\build
+          dir /S /B ..\build\vcpkg_installed\${{ matrix.config.VCPKG_TARGET_TRIPLET }}
+          dir /S /B ..\install
+
+      - name: ✔️❌✔️ Test
+        run: |
+          cd ..\install\tests
+          ttauri_tests.exe --gtest_output=xml:test_results_gtest.xml
+
+      - name: ♻️ Transform Test Results (gtest ➔ junit)
+        shell: pwsh
+        run: |
+          cd ..\install\tests
+          $xsl = New-Object System.Xml.Xsl.XslCompiledTransform
+          $xsl.Load("${{ github.workspace }}\.github\gtest_to_junit.xsl")
+          $xsl.Transform((Resolve-Path .\test_results_gtest.xml), "${{ github.workspace }}\test_results.xml")
+          cd ${{ github.workspace }}
+          ren test_results.xml test_results_${{ matrix.config.VCPKG_TARGET_TRIPLET }}_PCH_${{ matrix.config.TT_BUILD_PCH }}.xml
+
+      # We upload multiple files into the same artifact file (zip).
+      # Each file is differently named by adding the job name of the matrix as a suffix.
+      # This enables the "Unit Test Result" display to show all individual test runs of the matrix.
+      # The tests are published after all matrix runs finished (from job: "publish-test-results").
+      - name: 🔼 Upload Test Results
+        uses: actions/upload-artifact@v2 # https://github.com/actions/upload-artifact
+        if: always()
+        with:
+          name: test-results
+          path: test_results_*.xml
+
+      # Build Artifact Name: ttauri-0.3.0-0cda6a2-MSVC19-x64-static-RelWithDebInfo
+      - name: ✏️ Fetch Versioning Data & Set Artifact Name
+        shell: pwsh
+        run: |
+          $NAME=$(jq -r .name vcpkg.json)
+          $VERSION=$(jq -r .version vcpkg.json)
+          $SHORT_HASH=$($env:GITHUB_SHA.substring(0,7))
+          $COMPILER="MSVC19"
+          $TRIPLET=$("${{ matrix.config.VCPKG_TARGET_TRIPLET }}".Replace("-windows",""))
```

Comment:
I rather keep -windows as part of the artifact especially when we get -linux versions.